src/osx/iphone/checkbox.mm

```diff
@@ -37,29 +38,43 @@ + (void)initialize
 class wxCheckBoxIPhoneImpl : public wxWidgetIPhoneImpl
 {
 public:
-    wxCheckBoxIPhoneImpl(wxWindowMac *wxpeer, UISwitch *v)
+    wxCheckBoxIPhoneImpl(wxWindowMac *wxpeer, UISwitch *v, wxStaticText *l)
     : wxWidgetIPhoneImpl(wxpeer, v)
     {
         m_control = v;
+        m_label = l;
     }
 
-    wxInt32  GetValue() const
+    wxInt32  GetValue() const override
     {
         return [m_control isOn] ? 1 : 0;
     }
 
-    void SetValue( wxInt32 v )
+    void SetValue( wxInt32 v ) override
     {
         [m_control setOn:v != 0 animated:NO];
     }
+
+    void Move(int x, int y, int width, int height) override
+    {
+        wxWidgetIPhoneImpl::Move( x, y, width, height );
+        m_label->SetPosition( wxPoint(x + 65, y+5) );  // TODO: align properly
```

Comment:
It's not just alignment, it's the horizontal offset too. We really should at least use `GetChar{Width,Height}()` here instead of hardcoding these values.